setup.py

```diff
@@ -49,25 +58,25 @@
 
 
 setup(
-    name="<PYPI_NAME>",
+    name="docs-build",
     use_scm_version=True,
     setup_requires=["setuptools_scm"],
-    description="""<PYPI_NAME>: <SHORT_DESCRIPTION>""",
+    description="""docs-build: Build Sphinx Documentation for ApeWorX plugins""",
     long_description=long_description,
     long_description_content_type="text/markdown",
     author="ApeWorX Ltd.",
     author_email="admin@apeworx.io",
-    url="https://github.com/ApeWorX/<REPO_NAME>",
+    url="https://github.com/ApeWorX/docs-build",
     include_package_data=True,
-    install_requires=[],
+    install_requires=["shibuya=22024.7.13",],
```

Comment:
Does single quote work here? I always thought it had to be double

```suggestion
    install_requires=["shibuya"],
```

But also - I was poking around GitHub. Most dependents of `shibuya` exclude any pin (hence the suggestion)